### ISSUE TITLE
Improvement: Please update OctoPrint repository details

### ISSUE THREAD

**Issue by LMS0815** [FormerLurker/ArcWelderPlugin#174]
Seems _OctoPrint Software Updater/Manager_  has issues verifying the version of your plugin and maintaining/verifying available updates.

![image](https://user-images.githubusercontent.com/8235214/110439204-77127180-80b7-11eb-9137-bff2da33305b.png)

browser.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36
connectivity.connection_check: 8.8.8.8:53
connectivity.connection_ok: true
connectivity.enabled: true
connectivity.online: true
connectivity.resolution_check: octoprint.org
connectivity.resolution_ok: true
env.hardware.cores: 4
env.hardware.freq: 1400
env.hardware.ram: 899948544
env.os.bits: 64
env.os.id: linux
env.os.platform: linux
env.plugins.pi_support.model: Raspberry Pi 3 Model B Plus Rev 1.3
env.plugins.pi_support.octopi_version: 0.18.0
env.plugins.pi_support.throttle_state: 0x0
env.python.pip: 21.0.1
env.python.version: 3.7.3
env.python.virtualenv: true
octoprint.safe_mode: false
octoprint.version: 1.5.3

**Comment by FormerLurker** [comment]
Yes, this is a known issue, and it has been resolved in the latest dev and rc branches.  I will release it once I've worked out a few final bugs.  The new algorithm is superior, but is more complicated, and I don't want to ruin any prints :)
